javascript-algorithms-and-data-structures/Learn-Intermediate-OOP-by-Building-a-Platformer-Game/script.js

```diff
@@ -1,39 +1,42 @@
-const startBtn = document.getElementById("start-btn");
-const canvas = document.getElementById("canvas");
-const startScreen = document.querySelector(".start-screen");
-const checkpointScreen = document.querySelector(".checkpoint-screen");
-const checkpointMessage = document.querySelector(".checkpoint-screen > p");
-const ctx = canvas.getContext("2d");
+const startBtn = document.getElementById('start-btn');
+const canvas = document.getElementById('canvas');
+const startScreen = document.querySelector('.start-screen');
+const checkpointScreen = document.querySelector('.checkpoint-screen');
+const checkpointMessage = document.querySelector('.checkpoint-screen > p');
+const ctx = canvas.getContext('2d');
 canvas.width = innerWidth;
 canvas.height = innerHeight;
 const gravity = 0.5;
-let activeCheckpointCollisionDetection = true;
+let isCheckpointCollisionDetectionActive = true;
+
+function proportionalSize(s) {
+  return innerHeight < 500 ? Math.ceil((s / 500) * innerHeight) : s;
+}
```

Comment:
I think something like size would be a better parameter name 

```suggestion
function proportionalSize(size) {
  return innerHeight < 500 ? Math.ceil((size / 500) * innerHeight) : size;
}
```